### ISSUE TITLE
Error when calling Mathf.Triangulate

### ISSUE THREAD

**Issue by mudbug-z** [endlesstravel/Love2dCS#94]
Nuget: Love2dCS 11.0.42:
Using Windows 32-bit application, .net 4.8, when I call Mathf.Triangluate, I get the following:

```
Managed Debugging Assistant 'FatalExecutionEngineError' 
  Message=Managed Debugging Assistant 'FatalExecutionEngineError' : 'The runtime has encountered a fatal error. The address of the error was at 0x746a86b5, on thread 0x119c. The error code is 0xc0000005. This error may be a bug in the CLR or in the unsafe or non-verifiable portions of user code. Common sources of this bug include user marshaling errors for COM-interop or PInvoke, which may corrupt the stack.'
```

Running as 64-bit application I get this:

```
System.AccessViolationException
  HResult=0x80004003
  Message=Attempted to read or write protected memory. This is often an indication that other memory is corrupt.
  Source=<Cannot evaluate the exception source>
  StackTrace:
<Cannot evaluate the exception stack trace>
```

In case some context is needed, here where I am doing the call:
```
        public Polygon(float[] points)
        {
            _points = points;

            //put points into a vector structure as required by Triangulate
            var pointsAsVectors = new Vector2[points.Length/2];
            var j = 0;
            for (var i = 0; i < points.Length; i+= 2)
            {
                pointsAsVectors[j].X = points[i];
                pointsAsVectors[j].Y = points[i + 1];
                j++;
            }

            var triangles = Mathf.Triangulate(pointsAsVectors);
            foreach (var tri in triangles)
            {
                var trianglePoints = new float[6];
                trianglePoints[0] = tri.a.X;
                trianglePoints[1] = tri.a.Y;
                trianglePoints[2] = tri.b.X;
                trianglePoints[3] = tri.b.Y;
                trianglePoints[4] = tri.c.X;
                trianglePoints[5] = tri.c.Y;
                _triangles.Add(trianglePoints);
            }
        } 

```

**Comment by endlesstravel** [comment]
Thanks for your report! 
I had reproduce your problem on my pc. i will fix that bug.

**Comment by endlesstravel** [comment]
The bug on `Mathf.Triangulate` has fixed in version `11.0.43` .

**Comment by mudbug-z** [comment]
Thanks for the super-fast response!  I got the new version and it works great!
